### DISCUSSION TITLE
[QUESTION]:ilogtail与logtail的关系是什么

### DISCUSSION THREAD

**Discussion by SimonLiuhd** [alibaba/ilogtail#668] [Help] [answered]
阿里的[Logtail](https://www.alibabacloud.com/help/zh/log-service/latest/logtail-collection)与iLogtail有什么关系？iLogtail是Logtail的社区版本吗？

**Comment by messixukejia** [comment, marked as answer]
https://github.com/alibaba/ilogtail/discussions/268

可以参见下这个。 ilogtail 一般就是所说的社区版，logtail一般就是所说的企业版。
